Makefile

```diff
@@ -15,7 +15,7 @@ get-deps-%:
 # updated, find/replace all other pinned versions.
 .PHONY: get-deps-pip
 get-deps-pip:
-	pip install torch==1.9.0
+	pip install torch==1.11.0
```

Comment:
Was this one missed or does it need to live separately on purpose?

Reply:
Ah, I wasn't sure if it was important to keep that due to the comment above. But yeah, `torch==1.11.0` is now in multiple of those requirements files (since they constitute separate tests). Should be fine to remove. However, without `extras-requires`, version bumping torch on one of these requirements without bumping the others will make the linter freak out. This might be confusing for devs going forward.

Reply:
To fix _that_, it might be nice to find all the duplicated requirements and pull them out into a single common.txt included into the individual files with some `-r`s.

Or, possibly better, we put all of the versions into a single global constraints file and then in the individual requirements files, just include a `-c globalconstraints` and no versions on the packages.

Either way, that's probably a problem for a separate PR. :)